ui/raidboss/data/06-ew/dungeon/another_sildihn_subterrane.txt

```diff
@@ -0,0 +1,179 @@
+320.0 "Chilling Duster" sync / 1[56]:[^:]*:Silkie:7763:/
+332.9 "Total Wash (enrage)" sync / 1[56]:[^:]*:Silkie:79E0:/
+338.1 "Total Wash (enrage)" sync / 1[56]:[^:]*:Silkie:79F9:/
+
+#~~~~~~~~~~~~~~~~~~~~~~#
+# Gladiator of Sil'dih #
+#~~~~~~~~~~~~~~~~~~~~~~#
+
+# -ii 766F 7670 7676 7677 77BF 7660
+
+1000.0 "--sync--" sync / 00:0839::The Trial of Might will be sealed off/ window 1000,1
+1006.8 "--sync--" sync / 14:[^:]*:Gladiator of Sil'dih:7671:/ window 1007.3,5
+1011.5 "Flash of Steel" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7671:/
+1023.6 "Specter of Might" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7673:/
+1038.3 "Rush of Might" sync / 1[56]:[^:]*:Gladiator of Sil'dih:765B:/
+1040.4 "Rush of Might" sync / 1[56]:[^:]*:Gladiator of Sil'dih:765C:/
+1042.8 "Specter of Might" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7673:/
+1057.5 "Rush of Might" sync / 1[56]:[^:]*:Gladiator of Sil'dih:765B:/
+1059.6 "Rush of Might" sync / 1[56]:[^:]*:Gladiator of Sil'dih:765C:/
+1062.0 "--sync--" sync / 1[56]:[^:]*:Gladiator of Sil'dih:6854:/
+1067.0 "Sculptor's Passion" sync / 1[56]:[^:]*:Gladiator of Sil'dih:766C:/
+1067.3 "Sculptor's Passion" sync / 1[56]:[^:]*:Gladiator of Sil'dih:79F3:/
+1082.4 "Mighty Smite" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7672:/
+1095.5 "Curse of the Fallen" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7674:/
+1097.6 "--sync--" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7639:/
+1110.2 "--sync--" sync / 1[56]:[^:]*:Gladiator of Sil'dih:77BF:/
+1113.6 "Echo of the Fallen" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7675:/
+1121.3 "Flash of Steel" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7671:/
+1131.5 "--sync--" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7639:/
+1136.6 "Hateful Visage" sync / 1[56]:[^:]*:Gladiator of Sil'dih:766E:/
+1141.7 "Wrath of Ruin" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7663:/
+1150.7 "Gold Flame and Silver Flame" sync / 1[56]:[^:]*:Hateful Visage:(766F|7670):/
+1152.3 "Nothing beside Remains" sync / 1[56]:[^:]*:Gladiator of Sil'dih:(768B|768C):/
+1161.4 "Flash of Steel" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7671:/
+1171.6 "Accursed Visage" sync / 1[56]:[^:]*:Gladiator of Sil'dih:768D:/
+1176.7 "Wrath of Ruin" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7663:/
+1185.7 "Gold Flame and Silver Flame" sync / 1[56]:[^:]*:Hateful Visage:(766F|7670):/
+1187.3 "Nothing beside Remains" sync / 1[56]:[^:]*:Gladiator of Sil'dih:(768B|768C):/
+1196.4 "Flash of Steel" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7671:/
+1206.6 "--sync--" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7639:/
+1212.7 "Curse of the Monument" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7666:/
+1219.9 "Sundered Remains" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7668:/
+1220.5 "Sundered Remains" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7668:/
+1221.1 "Sundered Remains" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7668:/
+1221.7 "Sundered Remains" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7668:/
+1222.3 "Sundered Remains" sync / 1[56]:[^:]*:Gladiator of Sil'dih:7668:/
```

Comment:
This probably should be a duration and not 5 abilities?

Did you run your log through test_timeline? I feel like this would have broken.

Reply:
I have not run through test_timeline. I have had to use fflogs for generating the timelines as the utility could not find starting time for the encounters (might be bug or user error).